synapse/cores/lmdb.py

```diff
@@ -671,3 +802,47 @@ def _subrangeRows(self, p_enc, first_val, last_val, limit, right_closed, do_coun
                 if limit is not None and count >= limit:
                     break
         return count if do_count_only else ret
+
+    def _getCoreType(self):
+        return 'lmdb'
+
+    def _getBlobValu(self, key):
+        key_byts = s_common.msgenpack(key.encode('utf-8'))
+        db = self._dbs.get(BLOB_STORE)
+        with self._getTxn() as txn:
+            cur = txn.cursor(db)  # type: lmdb.Cursor
+            ret = cur.get(key_byts, default=None)
```

Comment:
You don't have to do the cursor thing unless you want to iterate:
    with self._getTxn() as txn:
        ret = txn.get(key_byts, default=None, db=db)